### PR TITLE
[v0.3 DeNas]refine CNN and VIT trainer

### PR DESCRIPTION
**Pull request by csdingbin**

### CODE REVIEW

File: e2eAIOK/common/trainer/data/data_builder_cv.py [hunk 1]
```diff
@@ -3,19 +3,32 @@
...
+        else:
+            raise RuntimeError(f"dataset {self.cfg.data_set} not supported")
+        return dataset_train, dataset_val
+    
+    def build_dataset(self):
+        transform = build_transform(is_train,cfg)
+        if cfg.data_set == 'CIFAR10':
+            dataset = datasets.CIFAR10(cfg.data_path, train=is_train, transform=transform, download=True)
+        elif cfg.data_set == 'CIFAR100':
+            dataset = datasets.CIFAR100(cfg.data_path, train=is_train, transform=transform, download=True)
```
Comment (Peach-He) [inline e2eAIOK/common/trainer/data/data_builder_cv.py:31]: for multi dataset, create data_builder_cifar10/cifar100 to support different datasets, and put common functions for CV in data_builder_cv

Comment (xuechendi) [reply e2eAIOK/common/trainer/data/data_builder_cv.py:31]: @Peach-He , the above comment is a todo or an explaination?

**Review by xuechendi — APPROVED** [review]
LGTM

File: e2eAIOK/common/trainer/data/data_builder_cv.py (hunk 1 above)
Comment (csdingbin) [reply e2eAIOK/common/trainer/data/data_builder_cv.py]: I think it don't need to provide many files separately for cifar10 and cifar100, it only need four lines to build the dataset.

Comment (xuechendi) [reply e2eAIOK/common/trainer/data/data_builder_cv.py]: > I think it don't need to provide many files separately for cifar10 and cifar100, it only need four lines to build the dataset.

will image1K and other dataset will be supported in future? If so, I think using a separate class is making sense

Comment (csdingbin) [reply e2eAIOK/common/trainer/data/data_builder_cv.py]: Got it. I will create a separate class for cifar dataset.

Comment (csdingbin) [reply e2eAIOK/common/trainer/data/data_builder_cv.py]: Resolved.

**Comment by csdingbin** [comment]
Conflicts had been resolved. Please help to merge @xuechendi

**Pull request merged**